src/fulltrade/FullTradeCard.js

```diff
@@ -57,11 +57,11 @@ export default class BaseTradeCard extends Component {
                     assetName={selectedSymbol}
                     onClosePanel={() => actions.removeTrade(index)}
                 />
-                {tick && <TradeChart
+                {(tick && Object.keys(tick).length > 0) && <TradeChart
                     symbol={selectedSymbol}
                     history={tick}
                 />}
-                <FullTradeParams {...this.props} />
+                {(contract && Object.keys(contract).length > 1) && <FullTradeParams {...this.props} />}
```

Comment:
If that's the case, how about undefined?
I believe there should never be a empty object though, empty object is `{}` right ?